plugins/cryptolib/cryptolib.hpp

```diff
@@ -66,6 +66,15 @@ CRYPTOLIB_API void CRYPTOLIB_CALL clPKISign(size32_t & __lenResult,void * & __re
 CRYPTOLIB_API bool CRYPTOLIB_CALL clPKIVerifySignature(const char * pkalgorithm,const char * publickeyfile,const char * passphrase,size32_t lenSignature,const void * signature,size32_t lenSigneddata,const void * signeddata);
 
 
+
```

Comment:
@jakesmith I do not know if anyone is using these features, but because it is documented and part of the STD library I think we need to go through the pains of deprecating

Reply:
I agree with Russ - (and Dan's comment in the Jira).  For the situation where a system only has one user (or doesn't even have any users configured), deploying the keys locally to the nodes makes a lot of sense, and doesn't introduce a security problem.
It also means the file can be read locally, rather than having to use a strange logical file syntax (which II suspect may still read remotely).

Reply:
right, hadn't seen or forgotten Dan's comment - I agree that's valid reason for wanting it.

But that means on systems that are multi user and they are trying to lock down access, the LDAP logical file validation can be bypassed by the versions that read from a file path.

Perhaps should instead allow direct paths, but validate users have permission via LDAP (if enabled) via another convention/ldap key, not via a LFN.

Reply:
So we will keep the local file system interface. I will remove all references to deprecation, and add to the Jira that we should elaborate on the risks of local keys, and highlight in the new DFS interfaces that users should lock them down with file scopes